src/reanimated2/js-reanimated/index.ts

```diff
@@ -31,8 +33,8 @@ export const _updatePropsJS = (
       [{}, {}]
     );
 
-    if (typeof component.setNativeProps === 'function') {
-      setNativeProps(component, rawStyles);
+    if (component.style !== undefined) {
+      updatePropsDOM(component, rawStyles);
```

Comment:
Did you test it on an older version of `react-native-web`? I'm just wondering about backward compatibility. I'm ok if it works only with the newest version of `react-native-web` but if it's true we need to give users information about that.

Reply:
It seems to work fine on `react-native-web@0.18.2` as well. Technically, we could have two implementations (new and legacy), one with `updatePropsDOM` (for `>=0.19`) and one with `setNativeProps` (for `<0.19`) but let's keep things simple.

Reply:
I've built the package and tested two examples (AnimatedStyleUpdateExample + transforms & SvgExample) across two Expo Web apps (with react-native-web 0.18.10 and 0.19.4) and everything seems to work fine so far.